README.md

```diff
@@ -15,14 +15,14 @@
 
 ## Summary
 
-- :zap: **Blazing fast performance:** running tasks in parallel using multiple processes.
-- :electric_plug: **Highly modular plugin system:** making it possible to change almost anything.
+- :zap: **Blazingly-fast performance:** running tasks in parallel using multiple processes.
```

Comment:
I think we better change it to **Blazing fast** (remove the performance) rather than Blazingly ... :)

Reply:
That's indeed better, haven't thought of it :)